kvbc/cmf/pruning_msgs.cmf

```diff
@@ -1,17 +0,0 @@
-# Represents a pruning agreement, i.e. when processing a PruneRequest.
-Msg Agreement 1 {
```

Comment:
Same as above, is it ok to remove it without removing concurrent turning?

Reply:
PR is also going to remove concurrent pruning code. I've updated title & description of PR.